backend/routes/client_reservation_routes.py

```
@@ -0,0 +1,117 @@
+import os
+from flask import Blueprint, jsonify, request, send_from_directory
+from sqlalchemy.orm import joinedload
```

Ruff (0.8.2)

`sqlalchemy.orm.joinedload` imported but unused

Remove unused import: `sqlalchemy.orm.joinedload`

(F401)